src/System.Diagnostics.Process/tests/ProcessStreamReadTests.cs

```diff
@@ -88,6 +90,120 @@ public void TestAsyncOutputStream()
             }
         }
 
+        [Fact]
```

Comment:
As we're explicitly sleeping for non-trivial periods of time, please mark this as [Outerloop]